README.md

```diff
@@ -15,254 +16,183 @@
+## **How to use Celo Composer**
 
-- Step 8: Follow the instructions to start the project. The same will be displayed on the console after the project is created.
+The easiest way to start is by using the `@celo/celo-composer` CLI tool. It helps bootstrap dApps with different templates.
 
-```text
-🚀 Your starter project has been successfully created!
+```bash
+npx @celo/celo-composer@latest create
 ```
 
-## Install Dependencies
-
+### **Installation Steps**
+1. **Enter a project name**
+2. **Choose whether to use Hardhat** (Yes/No)
+3. **Select a template** (Minipay, Valora, etc.)
+4. **Provide project owner details**
+5. **Wait for project creation to complete**
+6. **Follow the on-screen instructions to start your project**
 
-Once your custom dApp has been created, just install dependencies, either with yarn:
+---
 
-```bash
-   yarn
-```
-
-If you prefer npm, you can run:
+## **Install Dependencies**
+Once the project is set up, install dependencies:
 
 ```bash
-   npm install
+yarn
+# or
+npm install
 ```
 
-## Deploy a Smart Contract
-
-Find the detailed instructions on how to run your smart contract in [packages/hardhat/README.md](./packages/hardhat/README.md).
+---
 
-For quick development follow these three steps:
+## **Deploy a Smart Contract**
+For detailed instructions, refer to [`packages/hardhat/README.md`](./packages/hardhat/README.md).
 
-1. Change `packages/hardhat/env.template` to `packages/hardhat/env` and add your `PRIVATE_KEY` into the `.env` file.
-2. Make sure your wallet is funded when deploying to testnet or mainnet. You can get test tokens for deploying it on Alfajores from the [Celo Faucet](https://faucet.celo.org/alfajores).
-3. Run the following commands from the `packages/hardhat` folder to deploy your smart contract to the Celo Testnet Alfajores:
+Quick steps:
+1. Rename `packages/hardhat/.env.template` to `packages/hardhat/.env` and add your `PRIVATE_KEY`.
+2. Ensure your wallet has test funds from the [Celo Faucet](https://faucet.celo.org/alfajores).
+3. Deploy the contract:
 
 ```bash
 npx hardhat ignition deploy ./ignition/modules/Lock.ts --network alfajores
 ```
 
-## Deploy your Dapp Locally
-
-Find the detailed instructions on how to run your frontend in the [`react-dapp` README.md](./packages/react-app/README.md).
-
-Before you start the project, please follow these steps:
-
-1. Rename the file:
-   packages/react-app/.env.template
-   to
-   packages/react-app/.env
+---
 
-2. Open the newly renamed .env file and add your WalletConnect Cloud Project ID from [WalletConnect Cloud](https://cloud.walletconnect.com/)
+## **Deploy Your Dapp Locally**
+Before starting your project, follow these steps:
 
-Once you've done that, you're all set to start your project!
+1. Rename `.env.template` to `.env` in `packages/react-app/`.
+2. Add your WalletConnect Project ID from [WalletConnect Cloud](https://cloud.walletconnect.com/).
```

Comment:
2. Add your WalletConnect Project ID from [WalletConnect Cloud](https://cloud.walletconnect.com/) in the .env file.